src/jit/lsra.h

```diff
@@ -1131,7 +1131,7 @@ class LinearScan : public LinearScanInterface
     int compareBlocksForSequencing(BasicBlock* block1, BasicBlock* block2, bool useBlockWeights);
     BasicBlockList* blockSequenceWorkList;
     bool            blockSequencingDone;
-    void addToBlockSequenceWorkList(BlockSet sequencedBlockSet, BasicBlock* block);
+    void addToBlockSequenceWorkList(BlockSet sequencedBlockSet, BasicBlock* block, BlockSet predSet);
     void removeFromBlockSequenceWorkList(BasicBlockList* listNode, BasicBlockList* prevNode);
```

Comment:
Maybe:
```
void addToBlockSequenceWorkList(const BlockSet& sequencedBlockSet, BasicBlock* block, BlockSet& predSet);
```
since sequencedBlockSet is const/unchanged, and predSet is changed. If you don't use `&` then for short form bitsets you'll pass by value. It turns out that doesn't (currently) matter, since they don't allocate memory and you only care about reducing memory, but it would be weird for short bitsets to be passed by value and long bitsets to be passed by reference, accidentally.